apps/wing/src/commands/compile.ts

```diff
@@ -20,6 +20,8 @@ export interface CompileOptions {
   readonly target: wingCompiler.Target;
   readonly plugins?: string[];
   readonly rootId?: string;
+  readonly value?: string;
```

Comment:
can you please add some docstrings here? (I know there weren't docstrings here but maybe now is a good opportunity...)

Reply:
I added docstrings to all the variables here, @eladb, maybe I overdid it with the `values`